zipline/data/loader.py

```diff
@@ -43,6 +43,8 @@
     (treasuries, 'treasury_curves.csv', 'www.federalreserve.gov'),
 }
 
+MARKET_DATA_DIR = 'zipline/resources/market_data'
```

Comment:
What do you think about making this relative to the zipline module, as we did in in [`security_list`](https://github.com/quantopian/zipline/blob/3b5031a82959371b6c303f2381a7c15acae34e5b/zipline/utils/security_list.py#L17-L17). That method also ensures that we use the correct path separator between subdirectories - not sure if that's an issue or not, given that the tests passed on appveyor.